File: Java/백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine().trim());
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(br.readLine().trim());
        }
        Arrays.sort(arr);
        StringBuilder sb = new StringBuilder();
        for (int x : arr) {
            sb.append(x).append('\n');
        }
        System.out.print(sb);
    }
}

File: Python/백준/Bronze/2750. 수 정렬하기/수 정렬하기.py
import sys
input = sys.stdin.readline

def main():
    n = int(input().strip())
    arr = []
    for _ in range(n):
        arr.append(int(input().strip()))
    arr.sort()
    # 출력
    out = '\n'.join(str(x) for x in arr)
    print(out)

if __name__ == "__main__":
    main()